ports/ntf-core/portfile.cmake

```
@@ -42,23 +44,14 @@ endfunction()
 fix_install_dir("lib" "opt_exc_mt")
 fix_install_dir("debug/lib" "dbg_exc_mt")
 
-# The ntf-core build installs CMake configs for both targets into share/ntf-core, which
-# find_package will not be able to find on its own. We use vcpkg_cmake_config_fixup to install
-# CMake configs into share/nts, and then move the ntc CMake configs into share/ntc with this
-# function.
-function(ntf_core_fixup_ntc_config)
-    set(nts_share "${CURRENT_PACKAGES_DIR}/share/nts")
-    set(ntc_share "${CURRENT_PACKAGES_DIR}/share/ntc")
-    file(GLOB ntc_configs "${nts_share}/ntc*.cmake")
-    file(MAKE_DIRECTORY "${ntc_share}")
-    foreach(ntc_config IN LISTS ntc_configs)
-        file(RELATIVE_PATH ntc_config_rel "${nts_share}" "${ntc_config}")
-        file(RENAME "${ntc_config}" "${ntc_share}/${ntc_config_rel}")
-    endforeach()
-endfunction()
-
 vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake" PACKAGE_NAME nts)
-ntf_core_fixup_ntc_config()
+file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/share/ntc")
+file(COPY "${CURRENT_PACKAGES_DIR}/share/nts/ntc/"
+     DESTINATION "${CURRENT_PACKAGES_DIR}/share/ntc/")
+file(COPY "${CURRENT_PACKAGES_DIR}/share/nts/nts/"
+     DESTINATION "${CURRENT_PACKAGES_DIR}/share/nts/")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/nts/ntc")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/nts/nts")
```

Instead of a copy and then a delete, could we just rename them into place?

```suggestion
file(RENAME "${CURRENT_PACKAGES_DIR}/share/nts"
     DESTINATION "${CURRENT_PACKAGES_DIR}/share/nts_original")
file(RENAME "${CURRENT_PACKAGES_DIR}/share/nts_original/ntc"
     DESTINATION "${CURRENT_PACKAGES_DIR}/share/ntc")
file(RENAME "${CURRENT_PACKAGES_DIR}/share/nts_original/nts"
     DESTINATION "${CURRENT_PACKAGES_DIR}/share/nts")
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/nts_original")
```

---

Modified